atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/TargetedSweeperTest.java

```diff
@@ -232,6 +232,7 @@ public void sweepWithSingleEntryUpdatesMetrics() {
 
         assertThat(metricsManager).hasTombstonesPutConservativeEqualTo(1);
         assertThat(metricsManager).hasSweepTimestampConservativeEqualTo(getSweepTsCons());
+        sweepQueue.updateLastSweptTs(CONSERVATIVE);
```

Comment:
The tests will be failing when we remove the filtering, but we can fix them in a nicer way. One option is to use the per-shard metrics and then check the correct shard; other options include using only one shard, or (I think best) making sure some progress is made on all shards, then updating and checking